src/nwb2bids/bids_models/_model_globals.py

```diff
@@ -1,4 +1,4 @@
-_VALID_ID_REGEX = r"^[A-Za-z0-9]+$"
```

Comment:
@candleindark OK, this change is more intended for the follow-ups, but doesn't hurt to have here

Reply:
'Follow-up' is actually adjacent and can reviewed any time: https://github.com/con/nwb2bids/pull/176